pkg/apis/crd/v1alpha2/types.go

```diff
@@ -213,11 +216,18 @@ type EgressSpec struct {
 	// If ExternalIPPool is non-empty, it can be empty and will be assigned by Antrea automatically.
 	// If both ExternalIPPool and EgressIP are non-empty, the IP must be in the pool.
 	EgressIP string `json:"egressIP,omitempty"`
+	// EgressIPs specifies multiple SNAT IP addresses for the selected workloads.
```

Comment:
We probably should check the uniqueness of the pools because:
1. Using same ExternalIPPools multiple times is meaningless
2. It may cause trouble to the implementation, e.g. we might use map to track pool -> ip allocation.

Reply:
Ok. Would you add comments for that?

Reply:
updated the comment of ExternalIPPools

Reply:
Should we say IPs must be in ExternalIPPools? And move "Entries with the same index in EgressIPs and ExternalIPPools are correlated." here?

Reply:
But we could also support IPs not allocated from pools. For example, users could get HA support when they have two Nodes that have public access and just use two Node IPs as the Egress IPs here.